formats/json-jackson/src/main/java/io/cloudevents/jackson/JsonFormat.java

```diff
@@ -47,7 +47,7 @@ public final class JsonFormat implements EventFormat {
     /**
      * JSON Data Content Type Discriminator
      */
-    private static final Pattern JSON_CONTENT_TYPE_PATTERN = Pattern.compile("^(application|text)\\/([a-zA-Z]+\\+)?json$");
+    private static final Pattern JSON_CONTENT_TYPE_PATTERN = Pattern.compile("^(application|text)\\/([a-zA-Z]+\\+)?json(;[\\s\\w\"!#$%&'*+.^_`{|}~=-]*)*$");
```

Comment:
I wonder if this regex can be simplified so that it simply matches anything following the ";" so something like `"(;.*)*$"` - my regex may be a bit rusty. ;-)

Reply:
Thanks, I was overdoing it :)